### DISCUSSION TITLE
Welcome to sACN Discussions!

### DISCUSSION THREAD

**Discussion by sstaub** [sstaub/sACN#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iz1kbp** [comment]
Hi to all, sorry in advance for my poor english.I have not found any other way to talk to you about a problem with your sacn library.Can you give my a way to post my question/problem?
